pynsee/localdata/get_local_data.py

```diff
@@ -16,17 +16,19 @@
 from pynsee.utils._create_insee_folder import _create_insee_folder
 from pynsee.utils._hash import _hash
 
+import logging
+logger = logging.getLogger(__name__)
 
 @lru_cache(maxsize=None)
 def _warning_nivgeo(nivgeo):
     if nivgeo == "DEP":
         nivgeo_label = "departements"
-        print(f"By default, the query is on all {nivgeo_label}")
+        logger.info(f"By default, the query is on all {nivgeo_label}")

```

Comment:
these should be warnings (cf. function name)

Reply:
Same comment as previous : true the levels have default values in function's definition, but that doesn't qualify for a warning (this is a matter of documentation/docstring).